source/extensions/filters/http/compressor/compressor_library_config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
And as I think more about this I wonder if what we should have is extensions/compression/. And have the compression libraries be first class extensions.

And move the zlib implementation of the compressor there.

Reply:
It makes sense to me, because we would like to be able to compile different compression libraries in and out. For example when we add brotli it will be beneficial to be able to compile out zlib.